examples/collector-exporter-node/metrics.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
-const { ConsoleLogger, LogLevel } = require('@opentelemetry/core');
+const { DiagConsoleLogger, DiagLogLevel, diag } = require('@opentelemetry/api');
 const { CollectorMetricExporter } = require('@opentelemetry/exporter-collector');
 // const { CollectorMetricExporter } = require('@opentelemetry/exporter-collector-grpc');
 // const { CollectorMetricExporter } = require('@opentelemetry/exporter-collector-proto');
 const { MeterProvider } = require('@opentelemetry/metrics');
 
+diag.setLogger(new DiagConsoleLogger());
+diag.setLogLevel(DiagLogLevel.DEBUG);
+
 const metricExporter = new CollectorMetricExporter({
   serviceName: 'basic-metric-service',
   // url: 'http://localhost:55681/v1/metrics',
-  logger: new ConsoleLogger(LogLevel.DEBUG),
```

Comment:
now that its merely a diagnostics logger, does it make sense to include them in the examples for end user ?

Reply:
I agree, as it should be (hopefully) an exception case when they would need to provide. I left this in as part of this initial PR (before the followup to removed Logger) for 2 reasons
- It needs to be documented somewhere of how they would use it (having it commented out as with other examples etc)
- I don't have the history on why this was included in the first place as the Logger was also being used as a DiagnosticLogger just not by name.

Would you prefer this to be removed now or as part of the follow up breaking change to remove Logger?

Reply:
it was "merely" a diagnostics logger before. it will/should be used in all the same places

Reply:
> Would you prefer this to be removed now or as part of the follow up breaking change to remove Logger?

I think part of a future PR.

> it was "merely" a diagnostics logger before. it will/should be used in all the same places

I would argue that examples are here to explain how to setup different components, not how to debug them, so i would be in favor of having a documentation on what to when user encounters issues that explain to use this API, WDYT ?

Reply:
>I would argue that examples are here to explain how to setup different components, not how to debug them, so i would be in favor of having a documentation on what to when user encounters issues that explain to use this API, WDYT ?

I would be very in favor of this, but I think that is a follow-up. Let's keep this as it is for now to not scope-creep the PR and create an issue to make a DEBUGGING.md doc later.

Reply:
We could do this as part of #1878 (the follow up Issue) or as a seperate.